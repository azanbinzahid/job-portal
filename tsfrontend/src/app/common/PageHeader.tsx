import React, { FC } from "react";
import { Jumbotron } from "react-bootstrap";

type Props = {
  title: string;
  subtitle?: string;
};

const PageHeader: FC<Props> = ({ title, subtitle }) => {
  return (
    <Jumbotron<React.ElementType> align="center">
      <h1> {title} </h1>
      <h3> {subtitle} </h3>
    </Jumbotron>
  );
};

export default PageHeader;
