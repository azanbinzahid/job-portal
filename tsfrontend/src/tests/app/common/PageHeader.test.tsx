import React from "react";
import { shallow } from "enzyme";
import PageHeader from "app/common/PageHeader";
test("renders the component", () => {
  const component = shallow(<PageHeader title="abc" />);
  expect(component).toMatchSnapshot();
});
