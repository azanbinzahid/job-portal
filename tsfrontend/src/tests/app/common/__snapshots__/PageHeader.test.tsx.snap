// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<Jumbotron
  align="center"
  fluid={false}
>
  <h1>
     
    abc
     
  </h1>
  <h3>
     
     
  </h3>
</Jumbotron>
`;
